rskj-core/src/main/java/co/rsk/net/sync/BlockConnectorException.java

```diff
@@ -0,0 +1,19 @@
+package co.rsk.net.sync;
+
+public class BlockConnectorException extends RuntimeException{
+    private final long blockNumber;
+    private final long childBlockNumber;
+    public BlockConnectorException(final long blockNumber, final long childBlockNumber) {
+        super(String.format("Block is not parent of child. Block number: %s Child number: %s", blockNumber, childBlockNumber));
```

Comment:
WDYT of updating the message to be "Block with number %s is not child's (%s) parent."